### ISSUE TITLE
Minor indentation misalignment in keyboard shortcuts help

### ISSUE THREAD

**Issue by cellio** [codidact/qpixel#1020]
https://meta.codidact.com/posts/287473

In the help box for keyboard shortcuts, the entry for `t` is missing a space of indentation (compared to the other entries).  This is really minor and not worth fixing on its own, but if somebody is working on this help box anyway, please fix this while you're at it.

**Comment by cellio** [comment]
There is another issue on meta about this help box, but I don't know quite understand what the issue is or how we could address it, so I'll just link it here: https://meta.codidact.com/posts/287471

**Comment by trichoplax** [comment]
> There is another issue on meta about this help box, but I don't know quite understand what the issue is or how we could address it, so I'll just link it here: https://meta.codidact.com/posts/287471

That Meta post was intended to point out that the keyboard shortcut <kbd>t</kbd> only opened a dialog in a post, not in a list of posts such as a category's posts page (where instead a red outline appeared around the first post in the list, but with no dialog). I've checked and can confirm that #1954 fixes this (in addition to the other Meta post that this issue was based on). Now pressing <kbd>t</kbd> in a list of posts opens the Keyboard Shortcuts dialog to show the possible actions, so both Meta posts can be marked "status-completed" once that pull request is merged.
